CHANGELOG.md

```diff
@@ -47,6 +47,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 ## Version 2.0.0
 
+- [#470]
```

Comment:
Could you please move this istem to the "Upcoming version"?

The current items in the "Version 2.0.0" are already released. I will rename this section to "Version 2.0.0-beta0 (released)" to make it clearer. If you have some others ideas of what may help, let me know.

Reply:
For now I updated the changelog in this way
![Screenshot from 2023-10-13 12-32-27](https://github.com/learningequality/kolibri-design-system/assets/13509191/9c395318-9c1a-49a3-b638-5be2112cc526)

Reply:
From my experience looking through dependency updates for many different JS packages - I have found the style of having changes only mentioned in alphas, betas, and rcs and not in the final release much more difficult to parse, especially when looking for breaking changes. I'd recommend not separating the changelog in this way, and see the beta releases as try outs for what the final release will be.

Reply:
That makes sense and will be ideal in the long term.

This is temporary only to help us now when we're in the transition towards re-introducing a stable release process. We already have beta with release notes so that we can start introducing and regression testing. Without them, it'd be difficult because we were postponing releases and accumulated lots of breaking changes. So release notes were created in advance and with that we will need to keep track for some time what other changes need to be eventually added.

This should get better with more frequent releases. With @marcellamaki and @thanksameeelian, we decided to cleanup the release notes from all betas and merge them for the final release as soon as it's ready so that should address it. On that occasion we can do one last changelog cleanup and merge all sections for v2.

cc @thanksameeelian @marcellamaki

Reply:
Alternatively, we could just add a comment to indicate to the changelog where the release notes end. I tried something like that already, but it seems it's hard to notice for devs. In any case, I'd be okay with any way of making it easier for us to track where the existing release notes end in the changelog, if you have any other ides.

Reply:
Will rebase now to move this.

Reply:
Thanks